tests/application/app.ts

```diff
@@ -27,14 +27,14 @@ deviceManager.config.engineCollections.device.mappings.properties["softTenant"]
 
 
 deviceManager.models.registerDevice("DummyTempPosition", {
-  decoder: new DummyTempPositionDecoder(),
+  decoder: new DummyTempPositionDecoder(app.log),
   metadataMappings: {
     serial: { type: "keyword" },
   },
 });
 
 deviceManager.models.registerDevice("DummyTemp", {
-  decoder: new DummyTempDecoder(),
+  decoder: new DummyTempDecoder(app.log),
```

Comment:
Ici, c'est la fonction registerDevice qui devrait venir renseigner l'instance du logger plutôt que de l'avoir dans le constructeur.
Comme ca tu est sur que tous les decodeurs auront leur logger initialisé, et que l'utilisateur n'oublira pas de la mettre

Reply:
Dacc, j'ai rajouté la propriété logger dans ModelDefinition pour qu'on puisse le passer direct dans la méthode registerDevice qui vient renseigner l'instance dans le decodeur. on peut de la même manière faire un this.logger.debug etc dans les méthodes du décodeur sauf dans le constructeur dcp
![image](https://github.com/kuzzleio/kuzzle-device-manager/assets/78204354/62eee6e4-c749-422d-8171-d7560e881dbb)